proxy/http/HttpServerSession.cc

```diff
@@ -101,13 +101,21 @@ HttpServerSession::enable_outbound_connection_tracking(OutboundConnTrack::Group
 VIO *
 HttpServerSession::do_io_read(Continuation *c, int64_t nbytes, MIOBuffer *buf)
 {
-  return server_vc->do_io_read(c, nbytes, buf);
+  if (server_vc) {
```

Comment:
I suppose it's just a style thing, but I prefer the single `return` with ternary style.